packages/server/src/routes/auth/auth.test.ts

```
@@ -2,19 +2,21 @@ import { afterAll, describe, expect, test, jest } from "@jest/globals";
 import { testClient } from "hono/testing";
 import { RegisterFields } from "@repo/zod-types";
 import { createId } from "@paralleldrive/cuid2";
-import { deleteDB } from "../tests/setup";
+import { DBTestSetup } from "../tests/setup";
 import { authRouter } from ".";
 
 const newUniqueDate = createId();
 const testEmail = newUniqueDate + "+validemailtest@email.com";
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)